src/main/webapp/resources/js/components/samples/sampleSlice.js

```diff
@@ -0,0 +1,9 @@
+import { createSlice } from "@reduxjs/toolkit";
+
```

Comment:
Are you using this anywhere?

Reply:
Not yet but will be used in the SampleMetadata component. I just added it as I was setting up the redux stuff for this refactor. If I end up not using it I will remove it in the next PR